### ISSUE TITLE
Create overleaf documentation of each trained model.

### ISSUE THREAD

**Issue by Oguzvuruskaner** [Oguzvuruskaner/calorimetry-shower-data-generation#1]
Model overview
Motivation
Results
Hyperparameters
Loss functions and Interpretation
Comments
Feedback
Extra Information
